src/CXXTests/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
-get_property(SPHERAL_CXX_LIBS GLOBAL PROPERTY SPHERAL_CXX_LIBS)
 
 add_subdirectory(DeviceTestLib)
 
-if(ENABLE_TESTS)
+if(ENABLE_TESTS AND NOT ENABLE_DEV_BUILD)
   blt_add_executable( NAME spheral_cuda_test
                       SOURCES Spheral_CUDA_Test.cc
                       DEPENDS_ON Spheral_CXX ${spheral_blt_cxx_depends})
```

Comment:
Can we make this work with DEV_BUILD? We should be able to get the list of C++ libraries and make them a dependency right?

Reply:
We might be able to reproduce what we do https://github.com/LLNL/spheral/blob/b4b6ec5a5795d5132bfb774e0d16876483473180/cmake/spheral/SpheralAddLibs.cmake#L212